environments/servlet/tests/jetty/pom.xml

```diff
@@ -314,9 +314,9 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludes>
-                                <exclude>%regex[.*servlet/test/(?!se).*]</exclude>
-                            </excludes>
+                            <includes>
```

Comment:
It's failing because we're still missing `context.setInitParameter()` on line 154 - see also my last comment:

> Also we have to set the init param inside if (container instanceof ContainerInstance) {} block below because Weld allows to hand over Weld and WeldContainer instances.